annotation-model/scripts/JSONtest.js

```diff
@@ -163,13 +163,14 @@ function JSONtest(params) {
               // accumulate the assertions - but only when level is 0
               var list = [] ;
 
+              var type = "";
               if (assertions) {
                 if (typeof assertions === "object" && assertions.hasOwnProperty('assertions')) {
                   // this is a conditionObject
                   if (level === 0) {
                     list.push(assertContents[assertIdx]);
                   }
-                  var type = assertContents[assertIdx].hasOwnProperty('assertionType') ? assertContents[assertIdx].assertionType : "must" ;
+                  type = assertContents[assertIdx].hasOwnProperty('assertionType') ? assertContents[assertIdx].assertionType : "must" ;
```

Comment:
Not sure what the code style/guidelines are for this project, but this (and the other 3) seem a bit long.

Mind tweaking?

Not a blocker. Just askin' :smile: 

Otherwise this is :+1: